test/new-e2e/tests/agent-subcommands/flare/flare_common_test.go

```diff
@@ -25,11 +26,39 @@ type baseFlareSuite struct {
 }
 
 func (v *baseFlareSuite) TestFlareDefaultFiles() {
-	flare := requestAgentFlareAndFetchFromFakeIntake(v.T(), v.Env().Agent.Client, v.Env().FakeIntake.Client(), agentclient.WithArgs([]string{"--email", "e2e@test.com", "--send"}))
+	fakeIntake := v.Env().FakeIntake.Client()
+	flareArgs := agentclient.WithArgs([]string{"--email", "e2e@test.com", "--send"})
+	flare, _ := requestAgentFlareAndFetchFromFakeIntake(v.T(), v.Env().Agent.Client, fakeIntake, flareArgs)
 
 	assertFilesExist(v.T(), flare, defaultFlareFiles)
 	assertFilesExist(v.T(), flare, defaultLogFiles)
 	assertFilesExist(v.T(), flare, defaultConfigFiles)
+	assertFilesExist(v.T(), flare, defaultMetadataFlareFiles)
+	assertFoldersExist(v.T(), flare, defaultFlareFolders)
+
+	assertFilesExist(v.T(), flare, nonLocalFlareFiles)
+	assertFilesExist(v.T(), flare, nonLocalMetadataFlareFiles)
+
+	assertLogsFolderOnlyContainsLogFile(v.T(), flare)
+	assertEtcFolderOnlyContainsConfigFile(v.T(), flare)
+
+	assertFileContains(v.T(), flare, "process_check_output.json", "'process_config.process_collection.enabled' is disabled")
+	assertFileNotContains(v.T(), flare, "container_check_output.json", "'process_config.container_collection.enabled' is disabled")
+	assertFileNotContains(v.T(), flare, "process_discovery_check_output.json", "'process_config.process_discovery.enabled' is disabled")
+}
+
+func (v *baseFlareSuite) TestLocalFlareDefaultFiles() {
```

Comment:
You probably need to remove the 3 last lines of this test as these files are not created when using --local

Reply:
yes done